pkg/flink/internal/controller/input_controller.go

```diff
@@ -218,17 +222,35 @@ func getUnixBindings() []prompt.Option {
 	return []prompt.Option{
 		prompt.OptionAddASCIICodeBind(
 			prompt.ASCIICodeBind{
-				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left
+				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left (ESC + b)
 				Fn:        prompt.GoLeftWord,
 			},
 			prompt.ASCIICodeBind{
-				ASCIICode: []byte{0x1b, 0x66}, // Alt/Option + Arrow Right
+				ASCIICode: []byte{0x1b, 0x66}, // Alt/Option + Arrow Right (ESC + f)
 				Fn:        prompt.GoRightWord,
 			},
 			prompt.ASCIICodeBind{
 				ASCIICode: []byte{0x1b, 0x7F}, // Alt/Option + Backspace
 				Fn:        prompt.DeleteWord,
 			},
+			prompt.ASCIICodeBind{
+				ASCIICode: []byte{0x1b, 0x64}, // ForwardDeleteWord (ESC + d)
+				Fn: func(buf *prompt.Buffer) {
+					buf.Delete(buf.Document().FindEndOfCurrentWordWithSpace())
+				},
+			},
+			prompt.ASCIICodeBind{
+				ASCIICode: []byte{0x1b, 0x75}, // UpCaseWord (Alt/Option + u)
```

Comment:
For this, the Opt as Metakey has to be enabled, right? We could add this to the comment?

Do you maybe know what's the behavior on windows? (if the user has to enable something for this to work)

Reply:
I rewrote the comment to use "ESC" instead of "Alt". I think that's the most platform-generic way of specifying the key.

> Do you maybe know what's the behavior on windows? (if the user has to enable something for this to work)

I would assume that none of the shortcuts that use Alt work by default on windows. I'm not sure though and can't really test it.